Cargo.toml

```diff
@@ -56,10 +59,13 @@ tap = "1.0.1"
 stable_deref_trait = "1.2.0"
 thiserror = { workspace = true }
 camino = { workspace = true }
+abomonation = { workspace = true}
+abomonation_derive = { git = "https://github.com/winston-h-zhang/abomonation_derive.git" }
```

Comment:
Should we pull this into a `lurk-lab` fork?

Reply:
Arecibo itself has to as well. I've included that change in the CI PR, let's start with stamping that PR before I come back to fix things here:
https://github.com/lurk-lab/arecibo/pull/4

Reply:
Ah, merged, I can now adjust this PR.